frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import { AccountCode } from '../../api/account';
+import * as aoppAPI from '../../api/aopp';
+import { subscribe } from '../../decorators/subscribe';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { Button, Select } from '../forms';
+
+interface State {
+}
+
+interface AoppProps {
+}
+
+interface SubscribedProps {
+    aopp?: aoppAPI.Aopp;
+}
+
+type Props = SubscribedProps & AoppProps & TranslateProps;
+
+class Aopp extends Component<Props, State> {
+    public readonly state: State = {
+        aopp: {
+            state: 'inactive',
+            accounts: [],
+            errorCode: '',
+            address: '',
```

Comment:
there is also callbackHost

![Screen Shot 2021-06-24 at 9 49 36 AM](https://user-images.githubusercontent.com/546900/123234145-fac36300-d4da-11eb-9231-2d657d26822b.png)

Reply:
fixed